### ISSUE TITLE
docker:build to use a temporary directory inside the ${basedir}/target to avoid multi-project build issues

### ISSUE THREAD

**Issue by jstrachan** [rhuss/docker-maven-plugin#97]
I've just hit an issue where a second docker image in a build got jars from a previous docker build :) 

It seems like the temporary folder docker:build is using doesn't take into account the current ${basedir} from the project module.

It'd be good to make the default use ${basedir}/target/docker/... or something?

**Comment by jstrachan** [comment]
I think we need to add ${basedir}/ to these properties
https://github.com/jstrachan/docker-maven-plugin/blob/master/src/main/java/org/jolokia/docker/maven/BuildMojo.java#L50-50
https://github.com/rhuss/docker-maven-plugin/blob/master/src/main/java/org/jolokia/docker/maven/BuildMojo.java#L45

**Comment by jgangemi** [comment]
i thought `$basedir` was always assumed if the path is relative

**Comment by jstrachan** [comment]
It doesn't seem to be in a multi-maven build; I can see the docker plugin using the exact output folder for every module ;)

**Comment by jgangemi** [comment]
yeah - i just noticed that myself but adding in `$basedir` where you suggested doesn't work but i have another idea.

**Comment by jgangemi** [comment]
ok - i have a fix for this. let me put together a pull request.

**Comment by jstrachan** [comment]
great thanks!

FWIW as a workaround I added this to my pom.xml:
```
        <docker.target.dir>${basedir}/target/docker-plugin</docker.target.dir>
```
https://github.com/fabric8io/quickstarts/blob/master/pom.xml#L79

which seems to be a work around

**Comment by jgangemi** [comment]
fixed - if you want to build a local copy, feel free to pull from my branch.

**Comment by jstrachan** [comment]
thanks @jgangemi

**Comment by rhuss** [comment]
fixed in 0.11.2
